### ISSUE TITLE
Error in page "Exercise - Access data from a Blazor component"

### ISSUE THREAD

**Issue by nbourre** [MicrosoftDocs/mslearn-interact-with-data-blazor-web-apps#4]
This [page](https://docs.microsoft.com/en-us/learn/modules/interact-with-data-blazor-web-apps/5-exercise-access-data-from-blazor-components) has some errors which renders the tutorial useless.

- At point 10 in part "Use the database to display pizzas", there is not such line with "Configure the HTTP request pipeline"

In all respect, **You are Microsoft.** Please double-check your tutorial before publishing them...

**Comment by Thebe01** [comment]
If I'm correct this issue concerns module Program.cs mentioned in unit 5 of the tutorial.

**Comment by nbourre** [comment]
I haven’t touched the tutorial since the error. I've abandoned learning blazor because of this...

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Thebe01** [comment]
Si cela vous tente de vous remettre à l'apprentissage de Blazor...
Trouvé les 3 lignes suivantes dans Program.cs et inséré le code requis juste après :
app.MapRazorPages();
app.MapBlazorHub();
app.MapFallbackToPage("/_Host");

Cela vous permettra de terminer l'unité 5.

**Comment by Diaz15h** [comment]
There is another error in the "Use a service to obtain data" section
`<h1>Choose your pizza</a> the closing tag must </h1>`

**Comment by BagusTesa** [comment]
I do wonder if the tutorial is written using AI, I noticed this particular part of the tutorial feels disjointed compared to the previous sections.

It would be far easier to start from cloning this repo as step 1 and Exercise 4 should have been about writing `PizzaSpecialService` instead to replace the hard coded pizza specials.

**Comment by pmarangoni** [comment]
This module "Interact with data in Blazor web apps" is pathetically terrible. It's really hard to believe that Microsoft has gotten this bad. All they care about is selling subscriptions to Azure.

How about updating/fixing your tutorials?  I think we're supposed to be up to .NET 8 at this point.

**Comment by pmarangoni** [comment]
> Exercise 4 should have been about writing `PizzaSpecialService` instead to replace the hard coded pizza specials.

Exactly! It's as if different people wrote different parts of this without checking what else was already written.
